ui/ui-expenses/src/store/drivers/pica.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env browser */
+import pica from 'pica/dist/pica'
+
+const makeCanvas = (img, coeff = 1) => {
```

Comment:
signature could be: `makeResizingCanvas (image) -> { coefficient, canvas }`
And all the logic about resizing is here